src/server/mod.rs

```diff
@@ -18,8 +29,7 @@ use github::api::GithubSession;
 use jira;
 use jira::api::JiraSession;
 use server::github_handler::GithubHandlerState;
-use std::sync::Arc;
-
+use server::redirect_service::RedirectService;
 
 pub fn start(config: Config) -> Result<(), String> {
```

Comment:
i wonder if this should move to something like `start.rs` instead of being shoved into `mod.rs`...